### PR TITLE
Add docs about browser extension and triggers

### PR DESCRIPTION
**Pull request by michaelcfine**
Added doc info about triggers
Updated/added info about the browser plugin existing

### CODE REVIEW

**Comment by cletustboone** [comment]
Is it intentional that all the content of the `docs/Overview.md` docs is inside of triple backticks? It doesn't appear so since I see markdown links in the content.

File: docs/Triggers.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+Example:
+```
+cd server
+npm run dev -- --triggers ./src/triggers --triggers /some/other/dir/with/triggers
+```
+
+
+# File Organization
+
+Within each directory you pass via `--triggers` command line arguments, you must create subdirectories whose names match Firebolt method names (e.g., `lifecycle.ready`, `device.id`, etc.) and within these subdirectories you must create JavaScript files named `pre.mjs` and/or `post.mjs` depending on which trigger(s) you are defining.
```
Comment (cletustboone) [inline docs/Triggers.md:26]: The module names are now capitalized if that makes any difference. So it's `Lifecycle.ready()` now.

Comment (michaelcfine) [reply docs/Triggers.md:26]: Good to know. I'm going to leave things as they are right now so the docs match with the specific version of the SDK I'm using, but when I go to upgrade to a new version of the SDK, I'll make this change too.

**Review by cletustboone — APPROVED** [review]
🚀

File: docs/Triggers.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+# The Context Object
+
+When Mock Firebolt invokes your triggers, it passes your function a context object as the first parameter that includes functions you can use (call) within your triggers.
+
+
+setTimeout
+: The standard setTimeout function; included for your convenience
+
+setInterval
+: The standard setInterval function; included for your convenience
```
Comment (cletustboone) [inline docs/Triggers.md:58]: `setInterval` and `setTimeout` functions are already in the global namespace. Why do they need to be on `ctx` for convenience?

**Pull request merged**